src/App.tsx

```diff
@@ -1,17 +1,21 @@
-import { SignInButton, SignedIn, SignedOut, UserButton } from "@clerk/clerk-react"
+import {
+  SignInButton,
+  SignedIn,
+  SignedOut,
+  UserButton,
+} from '@clerk/clerk-react';
 
 function App() {
-
   return (
     <div>
       <SignedOut>
         <SignInButton />
       </SignedOut>
       <SignedIn>
-        <UserButton afterSignOutUrl="/" />
+        <UserButton />
```

Comment:
I think we should keep the `afterSignOutUrl` prop and PR the Quickstart guide to match.

Reply:
I rather change the quickstart, this prop is deprecated. I can do that first thing Monday

Reply:
I forgot this prop was moved to `<ClerkProvider />`. Let's move it there in both this repo and the guide. There is deffinitely value in including it.

Reply:
Done and related Docs PR https://github.com/clerk/clerk-docs/pull/1269